core/HalfLife2.cpp

```diff
@@ -326,15 +336,25 @@ bool UTIL_FindInSendTable(SendTable *pTable,
 	{
 		prop = pTable->GetProp(i);
 		pname = prop->GetName();
+		SendTable *pInnerTable = prop->GetDataTable();
 		if (pname && strcmp(name, pname) == 0)
 		{
+			// get true offset of CUtlVector
+			if (prop->GetOffset() == 0 && pInnerTable && pInnerTable->GetNumProps() && strncmp(pInnerTable->GetProp(0)->GetName(), "lengthproxy", 11) == 0)

```

Comment:
```suggestion
			if (prop->GetOffset() == 0 && pInnerTable && pInnerTable->GetNumProps() && strcmp(pInnerTable->GetProp(0)->GetName(), "lengthproxy") == 0)
```

It's matching the full string, so probably just want `strcmp` here?